lib/api/graphql/userDocument.dart

```diff
@@ -0,0 +1,9 @@
+String readUsers = """
```

Comment:
let's make this a method call instead because we want to be explicit about the variables we're inserting. The `QueryOptions` method allows you to insert variables when building it, but it doesn't enforce the number and type so we could run into runtime errors

```
String queryUser(String name, String username, String email) => """
    query user {
       user(id : "$id") {
         name
         id
         email
       } 
    }
""";

```